scripts/verifyWindows.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
This isn't meant to do anything different from verify.sh. The only change was adding "$SHELL" so that the pop up window doesn't close when I run this script. I can remove it if you think it's better.

I don't want to put in the time right now into adding a script for windows when this works fine.